models/issue_milestone.go

```diff
@@ -10,8 +10,8 @@ import (
 	"code.gitea.io/gitea/modules/setting"
 	api "code.gitea.io/gitea/modules/structs"
 	"code.gitea.io/gitea/modules/timeutil"
-	"xorm.io/builder"
 
+	"xorm.io/builder"
```

Comment:
```suggestion
```

Reply:
still used, this is only about code format

Reply:
cool. sorry I just wanted to make it easy to remove it if necessary!